content/about/contribute/github/index.md

```diff
@@ -28,30 +28,78 @@ The documentation is published under the [Apache
+There are three ways you can contribute to the Istio documentation:
 
 * If you want to edit an existing page, you can open up the page in your
   browser and select the **Edit This Page on GitHub** option from the gear menu
   at the top right of each page. This takes you to GitHub to edit and
   submit the changes.
 
-* If you want to work on the site in general, you must create a fork of the
-  repository. Click the button below to visit the GitHub repository. Then, you
-  must click the **Fork** button in the upper-right corner of the screen to
-  create a copy of our repository in your GitHub account. Create a clone of
-  your fork and make any changes you want. When you are ready to send those
-  changes to us, push the changes to your fork, go to the index page for your
-  fork, and click **New Pull Request** to let us know about it.
+* If you want to work on the site in general, follow the steps in our
+  [How to add content section](#add).
 
-<a class="btn btn-istio"
-href="https://github.com/istio/istio.io/">Browse this site's source
-code</a>
+* If you want to review an existing pull request (PR), follow the steps in our
+  [How to review content section](#review)
 
 Once your changes are merged, they show up immediately on
 `preliminary.istio.io`. However, the changes only
 show up on `istio.io` the next time we produce a new
 release, which happens around once a quarter.
 
+### How to add content {#add}
+
+To add content you must create a fork of the repository and a PR from
+your fork to the docs main repository. The following steps describe the
+process:
+
+1.  Click the button below to visit the GitHub repository.
+
+    <a class="btn btn-istio"
+    href="https://github.com/istio/istio.io/">Browse this site's source
+    code</a>
+
+1.  Click the **Fork** button in the upper-right corner of the screen to
+    create a copy of our repository in your GitHub account.
+
+1.  Create a clone of your fork and make any changes you want.
+1.  When you are ready to send those changes to us, push the changes to your
+    fork.
+1.  Go to the index page for your fork, and click **New Pull Request** to let
+    us know about it.
+
+### How to review content {#review}
+
+If your review is small, simply comment on the PR directly. If you review the
+content in detail, follow these steps:
+
+1.  Leave a comment on the PR with the text `/hold`. This command prevents the
+    PR from being merged before you are able to complete your review.
+
+1.  Perform your detailed review. When possible leave specific comments
+    directly on the files and lines affected.
+
+1.  Provide suggestions to the PR owner in your comments when appropriate. For
+    example:
+
+    {{< text markdown >}}
+    Use present tense to avoid verb congruence issues:
+
+    ```suggestion
+
+    Pilot maintains an abstract model of the mesh.
+
+    ```
+    {{< /text >}}
+
+1.  Publish your review to share your comments and suggestions with us and the
+    PR owner.
```

Comment:
I think that the `/hold cancel` should happen now. If you requested changes in your review, that will prevent the PR from merging.

Reply:
Will it? I've seen examples where it hasn't.

Reply:
yes it will prevent changes from merging if you click the "request changes button".  We generally don't use this button though, as the red turns away contributors.  Instead, we leave comments and reviewers don't merge the PR until the comments are resolved.

This solves a bit of a social problem (the negativity of the "request changes in Red" vs just guiding the committer towards a successful merge.

I suspect /hold and /hold cancel do the same thing.  As a reviewer, when I see a /hold, I take a good hard look at every review comment and change and am more thorough in the review.  Since /holds are so infrequent, it is warranted to be extra cautious.

I'd recommend removing hold from the workflow for this reason.

Cheers
-steve

Reply:
@sdake Sorry but I am not sure I followed. In your opinion: should the /hold be removed as soon as the **request changes button** is selected by the reviewer when they publish the review or after the all the comments and changes have been addressed?

Reply:
I think pressing the "request changes button" is counter to sending a positive message to contributors.  It shows up as a big red X.  Other reviewers will not approve a review until all relevant comments have been addressed.  I would advocate a workflow that doesn't expect reviewers to use hold/request changes features of github - but instead put a little extra work on reviewers to ensure all relevant issues are resolved.  Reviewers should be doing this work anyway, especially folks in the OWNERS file.

Reply:
@sdake I agree with your sentiment regarding sending a positive message to contributors. However, we need a process that scales well for both contributors and reviewers. I do like the use `/hold` over the request changes because it does not show a big red `X` beside the reviewers and because it ensures that no one inadvertently merges a contribution that is not approved by the reviewer. Historically, that "little extra work" on part of the reviewers has not happened and as the number of contributors grow, it becomes a lot harder for owners to keep track of both reviewers and contributors.

Reply:
I think that whether or not "request changes" looks like it's sending a positive message, it is the standard github process, so we should be following it. I think that `/hold` is a good way to temporarily block automatic merging until a review is complete. If that review is asking for changes, then marking the review as "request changes" is the proper way to control the merge at that point. Of course, if either the committer or reviewer is not responding quickly enough, and the PR is basically done and important enough, the owners/administrators can always go ahead and override and merge the PR at their discretion. Additional changes/fixes, if necessary, can then be done in a followup PR. This is one of the nice things about the way we initially publish to preliminary.istio.io, so it's not really live until the next release goes out.

Reply:
@frankbu You then agree that the `hold cancel` comments should happen once the review is published with **request changes** but before the changes are addressed?
I agree that the Owners can always override and merge at their discretion, that remains unchanged. Additional changes in subsequent PRs are also a possibility but keep in mind that we cannot force contributors to make them. 
@frankbu @sdake @geeknoid If you all agree that the `hold cancel` should happen right after the review is published, I'll change the steps to reflect it. What do you think?